app/controllers/Subscriptions.scala

```diff
@@ -0,0 +1,41 @@
+package controllers
+
+import actions.CustomActionBuilders
+import assets.AssetsResolver
+import com.gu.i18n.CountryGroup._
+import config.StringsConfig
+import play.api.mvc._
+import utils.RequestCountry._
+
+import scala.concurrent.ExecutionContext
+
+class Subscriptions(
+    actionRefiners: CustomActionBuilders,
+    val assets: AssetsResolver,
+    components: ControllerComponents,
+    stringsConfig: StringsConfig
+)(implicit val ec: ExecutionContext) extends AbstractController(components) {
+
+  import actionRefiners._
+
+  implicit val ar = assets
+
+  def subscribeGeoRedirect: Action[AnyContent] = GeoTargetedCachedAction() { implicit request =>
```

Comment:
Since it is inside the new subscriptions controller, maybe we can drop subscribe of each method.